circuits/precommit_checker.circom

```diff
@@ -0,0 +1,121 @@
+pragma circom 2.1.0;
+
+include "circomlib/poseidon.circom";
+include "circomlib/circuits/comparators.circom";
+include "circomlib/circuits/mimcsponge.circom";
+
+// include "https://github.com/0xPARC/circom-secp256k1/blob/master/circuits/bigint.circom";
+
+// check if m*G = precomp, and hash(m*G) = hash(precomp)
```

Comment:
Either add a space after `//` or don't — not both. When writing code the consistency is the key.